src/coreclr/vm/interpexec.cpp

```diff
@@ -68,28 +68,29 @@ std::invoke_result_t<Function> CallWithSEHWrapper(Function function)
     return local.result;
 }
 
+struct UnmanagedMethodWithTransitionParam
+{
+    MethodDesc *targetMethod;
+    int8_t *stack;
+    InterpMethodContextFrame *pFrame;
+    int8_t *pArgs;
+    int8_t *pRet;
+    PCODE callTarget;
+};
+
 // Use the NOINLINE to ensure that the InlinedCallFrame in this method is a lower stack address than any InterpMethodContextFrame values.
 NOINLINE
-void InvokeUnmanagedMethodWithTransition(MethodDesc *targetMethod, int8_t *stack, InterpMethodContextFrame *pFrame, int8_t *pArgs, int8_t *pRet, PCODE callTarget)
+void InvokeUnmanagedMethodWithTransition(UnmanagedMethodWithTransitionParam *pParam)
```

Comment:
Can you please add a brief comment explaining why we are passing the args via this struct instead of directly so that people can understand the reason in the future?

Reply:
I added a comment. If it isn't good enough, I will update again. Thank you.